src/bluesearch/entrypoint/database/parse.py

```diff
@@ -162,14 +166,30 @@ def run(
     """
     from bluesearch.utils import find_files
 
-    try:
-        inputs = find_files(input_path, recursive, match_filename)
-    except ValueError:
-        logger.error(
-            "Argument 'input_path' should be a path "
-            "to an existing file or directory!"
-        )
-        return 1
+    if input_path is None:
+        if sys.stdin.isatty():
+            # Real terminal session
+            logger.error("No input files provided")
+            return 1
+        else:
+            # Piped session
+            input_lines = sys.stdin.read().split()
+            inputs = []
+
+            for line in input_lines:
+                path = Path(line)
+                if path.exists():
+                    inputs.append(path)
```

Comment:
Do you think we should inform the user when the path does not exist ? Normally it should never happen, right ?

Reply:
I wondered about the same thing!